apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -169,26 +172,34 @@ export const FluentDocsPage = () => {
             <Description />
             {videos && <VideoPreviews videos={videos} />}
           </div>
-          <hr className={styles.divider} />
-          <HeaderMdx as="h3" id={nameToHash(primaryStory.name)}>
-            {primaryStory.name}
-          </HeaderMdx>
-          <Primary />
-          <ArgsTable of={primaryStory.component} />
-          {primaryStory.argTypes.as && primaryStory.argTypes.as?.type?.name === 'enum' && (
-            <div className={styles.nativeProps}>
-              <InfoFilled className={styles.nativePropsIcon} />
-              <div className={styles.nativePropsMessage}>
-                <b>
-                  Native props are supported <span role="presentation">🙌</span>
-                </b>
-                <span>
-                  All HTML attributes native to the {getNativeElementsList(primaryStory.argTypes.as.type.value)},
-                  including all <code>aria-*</code> and <code>data-*</code> attributes, can be applied as native props
-                  on this component.
-                </span>
-              </div>
-            </div>
+          {skipPrimaryStory ? null : (
```

Comment:
discussed offline: it seems that this is needed only for special purposes of motion stories, which are not adhering to our SB patterns for authoring stories. We should follow our practices and if there is something that needs to change it should be discussed with others and documented(ideally automated). I dunno who introduced these new patterns for motion stories so cc-ing @layershifter ty